content/post/kubedb-v2024.06.04/index.md

```diff
@@ -0,0 +1,781 @@
+  version: "3.13.2"
+  replicas: 3
+  storage:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+    storageClassName: standard
+  serviceTemplates:
+  - alias: primary
+    spec:
+      type: LoadBalancer
+  enableSSL: true
+  tls:
+    issuerRef:
+      apiGroup: "cert-manager.io"
+      kind: Issuer
+      name: rabbitmq-ca-issuer
+  storageType: Durable
+  deletionPolicy: Delete
+```
+
+**New Version support**: `3.13.2`.
+
+## SingleStore
+
+### OpsRequest
+In this release, we have introduced support for SingleStore Ops Requests. Initially, Ops Requests for Restart, Vertical Scaling, Volume Expansion, and Reconfiguration have been added for both clustering and standalone modes.
+
+#### Vertical Scaling
+Vertical Scaling allows you to vertically scale the SingleStore nodes (i.e., pods). The necessary information for vertical scaling must be provided in the `spec.verticalScaling.(aggregator/leaf/node/coordinator)` field.
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: SinglestoreOpsRequest
+metadata:
+  name: sdb-vscale
+  namespace: demo
+spec:
+  type: VerticalScaling  
+  databaseRef:
+    name: sdb-sample
+  verticalScaling:
+    aggregator:
+      resources:
+        requests:
+          memory: "2500Mi"
+          cpu: "0.7"
+        limits:
+          memory: "2500Mi"
+          cpu: "0.7"
+
+```
+#### Volume Expansion
+Volume Expansion allows you to expand the storage of the SingleStore nodes (i.e., pods). The necessary information for volume expansion must be provided in the `spec.volumeExpansion.(aggregator/leaf/node)` field.
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: SinglestoreOpsRequest
+metadata:
+  name: sdb-volume-ops
+  namespace: demo
+spec:
+  type: VolumeExpansion
+  databaseRef:
+    name: sdb-sample
+  volumeExpansion:
+    mode: "Offline"
+    aggregator: 10Gi
+    leaf: 20Gi
+```
+#### Reconfiguration
+Reconfiguration allows you to update the configuration through a new secret or apply a config. Users can also remove the custom config using RemoveCustomConfig. The necessary information for reconfiguration must be provided in the `spec.configuration.(aggregator/leaf/node)` field.
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: SinglestoreOpsRequest
+metadata:
+  name: sdbops-reconfigure-config
+  namespace: demo
+spec:
+  type: Configuration
+  databaseRef:
+    name: sdb-standalone
+  configuration:
+    node:
+      applyConfig:
+        sdb-apply.cnf: |-
+          max_connections = 350
+```
+### Autoscaler
+
+In this release, we are also introducing the SinglestoreAutoscaler, a Kubernetes Custom Resource Definition (CRD) that supports autoscaling for SingleStore. This CRD allows you to configure autoscaling for SingleStore compute resources and storage in a declarative, Kubernetes-native manner.
+
+Deploying the SingleStore Autoscaler
+
+To deploy an Autoscaler for a KubeDB-managed SingleStore cluster, you can use the following YAML configuration:
```

Comment:
Add a short description about the autoscaler yaml. like what it does when scalers will be triggered etc.